src/main/java/ast/psi/DelegateStoreImpl.java

```diff
@@ -0,0 +1,27 @@
+package ast.psi;
```

Comment:
A 'delegate' object is the PsiElement from the IntelliJ SDK that we "delegate" actions to from the wrapper classes ending in the word 'delegate'